test/tsconfig.json

```diff
@@ -4,13 +4,14 @@
     "composite": true,
     "outDir": "./lib",
     "tsBuildInfoFile": ".tsbuildinfo",
-    "rootDir": "./src",
+    "rootDir": "../",
```

Comment:
is this file still necessary here, can it be removed or put somewhere else?

Reply:
Seems like this is still a reasonable place for it, since it's the tsconfig for tests. It could be at the top level, but this seems fine too?